docs/quick-start.rst

```diff
@@ -0,0 +1,115 @@
+Quick Start
+===========
+
+Database Setup
+--------------
+
+.. code-block:: bash
+
+    docker run -it -p 27017:27017 mongo
+
+
+Service Setup
+-------------
+
+.. code-block:: bash
+
+    npm install graphql-auth-service --save
+
+.. code-block:: js
+
+    const { GraphQLAuthService } = require('graphql-auth-service');
+    const express = require('express');
+    
+    const app = express();
+    
+    app.use('/auth', GraphQLAuthService());
+    
+    app.listen(process.env.PORT || 5000, () => {
+        console.log(`server is listening on ${process.env.PORT || 5000}`)
+    })
+
+.. code-block:: bash
+
+    curl -X POST -H "Content-Type: application/json" -d '{"query":"{ publicKey }"}' \
+        localhost:5000/auth
+    
+.. _graphql-queries:
+
+GraphQL Queries
+---------------
+
+To use GraphQL Auth Service, you can use the ``fetch`` method or the ``XMLHttpRequest`` Object in JavaScript. To make an authenticated request, simply include your authentication token as ``Bearer token`` in the ``Authorization`` header of your request. Please refer to this example below:
+
+.. code-block:: js
+   
+    // GraphQL query
+    let query =
+        `mutation {
+            updateMe(fields: {username:"newusername"}) {
+                token
+                notifications{
+                    message
+                }
+            }
+        }`;
+
+    fetch(yourServiceURL, {
+            method: 'post',
+            headers: {
+                'Content-Type': 'application/json',
+                // Authentication token to make an authenticated request
+                'Authorization': 'Bearer ' + yourAuthToken
+            },
+            body: JSON.stringify({
+                query: query
+            }),
+        })
+        .then(res => res.json())
+        .then(res => console.log(res));
```

Comment:
let headers = {
    'Content-Type': 'application/json',
    'Authorization': 'Bearer ' + yourAuthToken // To make an authenticated request
};
let query =
    `mutation {
        updateMe(fields: {username:"newusername"}) {
            notifications{
                message
            }
        }
    }`;
let body = JSON.stringify({ query });

fetch('http://your-service-url.com', { method: 'post', headers, body })
.then(res => res.json())
.then(res => console.log(res));